assets/js/components/Dashboard/PlaceRow.tsx

```diff
@@ -151,6 +151,25 @@ const PlaceRow = (props: PlaceRowProps): JSX.Element => {
     )?.inlineMap;
   };
 
+  const formatStationName = (stationName: string) => {
+    let isTerminalStop = false;
+    if (props.filteredLine && props.filteredLine !== null) {
+      const line = STATION_ORDER_BY_LINE[props.filteredLine.toLowerCase()];
+      const terminalStops = line.filter((line) => line.isTerminalStop);
+      isTerminalStop = terminalStops.some((stop) => stationName === stop.name);
+    } else {
+      isTerminalStop = Object.keys(STATION_ORDER_BY_LINE)
+        .filter((key) => !["silver", "mattapan"].includes(key))
```

Comment:
Why weed out these two routes?

Reply:
Mostly just because they are unnecessary lookups. Nothing will return true as a terminal stop in these in the current code. For SL, I think that makes sense because we do not show the SL in its entirety in the place list. Does that make sense for Mattapan? Do we have terminal stops for that line? I know there is a start and end to it but I wasn't sure if they are considered that in the same way subway does.

Reply:
No matter what, every place row requires exactly 1 lookup to decide whether its name should be capitalized or not, right? The lookup should just be "am I a terminal stop?" And this code gets executed for every single PlaceRow individually, so no place row is aware of the "terminal-ness" of the other rows. I'm not sure how this conditional prevents any lookups.

Reply:
We can only reliably look in the list for a single route if a route filter is active. Just knowing if a stop is a terminal stop is not quite enough when a filter is applied. For Gov't Center, it is a terminal stop on the GL but not on the BL. If we went with the "am I a terminal stop?" approach, it would be capitalized for BL when it should not be. That's the primary reason for the conditional. So when no filter is applied, we find if a stop is a terminal stop for any line. If a filter is applied, we find if a stop is a terminal stop for only the line in the filter.

Reply:
OHHH! Multi-route stops totally went over my head. Gov't Center was the perfect example, thank you